src/components/command-bar/commands/search/components/dialog-body/index.tsx

```diff
@@ -68,37 +92,25 @@ const SearchCommandBarDialogBody = () => {
 	 */
 	useSetUpAndCleanUpCommandState(setUpCommandState, cleanUpCommandState)
 
+	/**
+	 * Get the CommandBarTag object for the current product if it's present.
+	 */
+	const currentProductTag = useMemo(
+		() =>
+			getCurrentProductTag({
+				currentProduct,
+				currentTags,
+			}),
+		[currentProduct, currentTags]
+	)
+
 	return (
-		<>
-			{currentInputValue ? (
-				<Tabs
-					showAnchorLine={false}
-					initialActiveIndex={contentType === 'tutorials' ? 1 : 0}
-				>
-					<Tab heading="Documentation">
-						<DocumentationTabContents
-							searchResults={searchResults.docs}
-							suggestedPages={suggestedPages}
-						/>
-					</Tab>
-					<Tab heading="Tutorials">
-						<TutorialsTabContents
-							searchResults={searchResults.tutorials}
-							tutorialLibraryCta={{
-								href: currentProductTag
-									? `/tutorials/library?product=${currentProductTag.id}`
-									: '/tutorials/library',
-								text: currentProductTag
-									? `See all ${currentProductTag.text} tutorials in the Tutorial Library`
-									: 'See all tutorials in the Tutorial Library',
-							}}
-						/>
-					</Tab>
-				</Tabs>
-			) : (
-				<SuggestedPages pages={suggestedPages} />
-			)}
-		</>
+		<InstantSearch indexName="prod_LEARN" searchClient={searchClient}>
```

Comment:
❓ question: I'm wondering why we need the learn `InstantSearch` wrapped at the top-level here? Are we using the learn algolia data outside of its tab?

Reply:
This is because `InstantSearch` has to be rendered to use the hooks, and `indexName` is a required prop. So I just went with `prod_LEARN` when I originally wrote this (before I knew we had just one index for docs) since it was an index that always needed to be used.

I updated to reference the `tutorialsIndexName` configuration variable though! Forgot to do that when I made the variable. Did that in https://github.com/hashicorp/dev-portal/pull/990/commits/3d8ddab830d853d24d184c9f6414c4b64bad6a40